### ISSUE TITLE
pybedtools and bedtools

### ISSUE THREAD

**Issue by sebastian-luna-valero** [bioconda/bioconda-recipes#5351]
Hi @daler,

I haven't looked at `pybedtools`' source code but I was wondering why its recipe does not depend on any version of `bedtools` either at build or run time? (the same way that `pysam` depends on `samtools`)
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pybedtools/meta.yaml

Best regards,
Sebastian

**Comment by daler** [comment]
@sebastian-luna-valero it's for purely historical reasons -- pybedtools is specifically written to be agnostic about the version of bedtools installed, so it's not part of the python packaging. Bedtools was always separately managed. Then pybedtools was one of the first bioconda recipes, and I carried that idea forward, not including bedtools as a dependency.

Then I just kept bumping the version number, without changing the deps without noticing bedtools wasn't a dep.

Anyway, bedtools should really be a dependency of the pybedtools conda package, thanks for pointing that out and I'll make the change.
